<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
      <item>
       <widget class="QTreeWidget" name="exp_menu">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="recv_tab">
         <attribute name="title">
          <string>Received WebRTC stats</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
        <widget class="QWidget" name="qlog_tab">
         <attribute name="title">
          <string>QLOG Stats</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
        <widget class="QWidget" name="medooze_tab">
         <attribute name="title">
          <string>Medooze stats</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
        <widget class="QWidget" name="all_tab">
         <attribute name="title">
          <string>All</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="legend_list"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuic_Tunnel_stats_viewer">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuQuic_Tunnel_stats_viewer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
